src/style/planet.css

```diff
@@ -0,0 +1,241 @@
+/* Regular theme starts */
```

Comment:
Did you `git mv` this file?  It should show it as a rename vs. a new file being created.

Reply:
I didn't do git mv and now I understand why I saw that new file was created...What should I do now? Reverse changes that was done?

Reply:
```
git rm src/style/planet.css
mkdir -p src/frontend/style
git mv planet.css src/frontend/style
```

NOTE: I've changed this to include a `src/frontend` parent dir to separate the frontend code from the server, parsers, etc.

Reply:
Thank you @humphd